src/m_misc.c

```diff
@@ -2483,6 +2491,13 @@ default_t defaults[] = {
     "color range used for chat messages and entry"
   },
 
+  { // gold range
+    "hudcolor_obituary",
+    (config_t *) &hudcolor_obituary, NULL,
+    {CR_GOLD}, {CR_BRICK,CR_NONE}, number, ss_mess, wad_yes,
```

Comment:
How about `CR_RED` as the default color? The gold color looks weird to me. Is it from ZDoom?

Reply:
Yes, gold is the default color from gzdoom. I don't like it either. Actually, i prefer gray instead, c.f. "obituary". 😉 

Enable by default may be a good idea. Usually we don't do this, but this way the feature may have a chance of getting recognized.

Reply:
Yeah, I agree. Gold is good for messages like "a secret is revealed" etc.